templates/docs/apis/_new.md

```diff
@@ -30,7 +30,7 @@ API applications have `actions/app.go` and `actions/render.go` files that are a
 <%= partial("docs/apis/api_app.md") %>
 <%= partial("docs/apis/api_render.md") %>
 
-<h5><code>buffalo new coke</code></h5>
+<h5><code>buffalo new pepsi</code></h5>
 
 <%= partial("docs/apis/web_app.md") %> 
-<%= partial("docs/apis/web_render.md") %>
\ No newline at end of file
+<%= partial("docs/apis/web_render.md") %>
```

Comment:
actually, this line shouldn't be updated.. sorry, I used the github web UI to make the change... let me remove this line